src/test/rpc/NoRipple_test.cpp

```diff
@@ -84,7 +84,7 @@ class NoRipple_test : public beast::unit_test::suite
 
         // fix1578 changes the return code.  Verify expected behavior
         // without and with fix1578.
-        for (auto const tweakedFeatures :
+        for (auto const& tweakedFeatures :
```

Comment:
Interesting. I found this one, but only this one, with VS 2019 over at https://github.com/ripple/rippled/pull/3851/commits/a51cfddc0aea40387d7ab578e68b6ebb7beee927#diff-9bec26b172c29e97751e1152a678d71aadb13f48bb3f62fe6962113dbdca7910. I doubt it'll cause any merge problems, but it's interesting.